fe/fe-core/src/main/java/org/apache/doris/analysis/SortInfo.java

```diff
@@ -117,6 +119,24 @@ public void setMaterializedTupleInfo(
         }
     }
 
+    /**
+     * Sets tupleInfo.
+     * Just for Nereids.
+     */
+    public void setTupleInfo(
```

Comment:
Wraps up as an implementation of the expr like ```finalizeImplForNereids``` function, and annotated only for the nereids optimizer translator.

Reply:
done